examples/ui-demo/src/components/preview/CodePreview.tsx

```diff
@@ -50,9 +52,41 @@ function CodeBlock({ title, code }: { title: string; code: string }) {
   );
 }
 
-const styleCode = `import { withAccountKitUi } from "@alchemy/aa-alchemy/tailwind";
+
+function getTailwindCode(config: Config) {
+  const { ui } = config
+  // TODO: separate primaryColor for light and dark mode
+  return `import { withAccountKitUi, createColorSet } from "@alchemy/aa-alchemy/tailwind";
 import type { Config } from "tailwindcss";
 
-const config: Config = <your tailwind config>;
-export default withAccountKitUi(config);
-`;
+export default withAccountKitUi(<your tailwind config>, {
+   // override account kit themes
+   colors: {
+     "btn-primary": createColorSet("${ui.primaryColor}", "${ui.primaryColor}"),
+     "fg-accent-brand": createColorSet("${ui.primaryColor}", "${ui.primaryColor}"),
+   },
+ })
+`
+}
+
+function getConfigCode(config: Config) {
+  const sections = []
+
+  if (config.auth.showEmail) {
+    sections.push([{ type: "email" }])
+  }
+
+  if (config.auth.showExternalWallets) {
+    sections.push([{ type: "injected" }])
+  }
+
+  sections.push([{ type: "passkey" }])
+
+  return `const uiConfig = {
+  auth: {
+    sections: ${JSON.stringify(sections)},
+    addPasskeyOnSignup: ${config.auth.addPasskey},
+  },
+};
+  `
```

Comment:
I think this should include the full code snippet including the `createConfig` call so maybe something like:

```
export const config = createConfig({
 // the default config we recommend in the examples for that method (think rpcUrl: /api/rpc and chain: sepolia
});
```